src/js/pages/services/ServicesTab.js

```diff
@@ -148,14 +164,27 @@ var ServicesTab = React.createClass({
           <div className="flex-grow">
             {breadcrumbs}
             {filterHeadline}
-            <ServiceSearchFilter
-              handleFilterChange={this.handleFilterChange} />
+            <FilterBar rightAlignLastNChildren={2}>
```

Comment:
This function is becoming huge. Could we break this into smaller functions?
I know not all of this was introduced in this PR, but it is extending it, so I think it makes sense to handle it up front